src/asm/rgbasm.5

```diff
@@ -399,6 +399,8 @@ Most of them return a string, however some of these functions actually return an
 .It Fn STRLWR str Ta Returns Ar str No with all letters in lowercase.
 .It Fn STRRPL str old new Ta Returns Ar str No with each non-overlapping occurrence of the substring Ar old No replaced with Ar new .
 .It Fn STRFMT fmt args... Ta Returns the string Ar fmt No with each
+.It Fn CHARLEN str Ta Returns the number of charmap entries in Ar str No with the current charmap.
+.It Fn CHARSUB str pos Ta Returns the substring for the charmap entry at Ar pos No in Ar str No Po first character is position 1 Pc with the current charmap.
```

Comment:
No need for `Po` and `Pc`, regular parens are fine.

Reply:
Oh, I was copying `STRSUB`'s line.